doc/dev/mgmt/tests.md

```diff
@@ -15,58 +15,43 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
-    or online, by authenticating with Azure to record new HTTP interactions
+-   How to run the tests offline (using previously recorded HTTP interactions)
+-   How to run the tests online (by authenticating with Azure to record new HTTP interactions)
 -   How to write new tests using our utility classes
 
 # Getting the tests to run
 
-This section describes how to run the SDK tests,
-by installing dependencies into a virtual environment
-and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner
-to choose which tests to run.
+This section describes how to run the SDK tests, by installing dependencies into a virtual environment and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner to choose which tests to run.
 
 ## Running the tests
 
-Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner.
-To run all the tests, you can use the following command:
+Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner. To run all the tests, you can use the following command:
 
 ```Shell
 pytest
 ```
 
-You can provide directories or individual files as positional arguments
-to specify particular tests to run
-rather than running the entire test suite. For example:
+You can provide directories or individual files as positional arguments to specify particular tests to run rather than running the entire test suite. For example:
 ```Shell
-pytest -s sdk/storage/azure-mgmt-storage/
+pytest sdk/storage/azure-mgmt-storage/
 pytest sdk/storage/azure-mgmt-storage/tests/test_mgmt_storage.py
 ```
 
-By default, tests run in playback mode,
-using recordings of HTTP interactions to simulate
-requests made against Azure and allow the tests to run offline.
-To run the tests in live (or "recording") mode,
-you'll need to set up token-based Azure authentication.
+By default, tests run in playback mode, using recordings of HTTP interactions to simulate requests made against Azure and allow the tests to run offline. To run the tests in live (or "recording") mode, you'll need to set up token-based Azure authentication.
 
 ## Getting Azure credentials
 
-There are several ways to authenticate to Azure,
-but to be able to record test HTTP interactions,
-you must use an OAuth authentication method which gives you a token:
+There are several ways to authenticate to Azure, but to be able to record test HTTP interactions, you must use an OAuth authentication method which gives you a token:
 - Azure Active Directory user/password
 - Active Directory application and service principal
 
 Certificate authentication does not allow you to record HTTP queries for testing.
 
-### Get a token with Azure Active Directory user/password. This is considered deprecated and should not be used anymore (https://docs.microsoft.com/python/azure/python-sdk-azure-authenticate?view=azure-python#mgmt-auth-legacy).
+<!-- If it's deprecated, do we want to have it in docs? -->
```

Comment:
Let's decide on this now and yank it or keep it, rather than leave in limbo.  Is anyone still relying on this such that we'd have compelling reason to maintain it?

Reply:
@lmazuel Is there a reason to keep this around?